modules/customize-certificates-add-service-serving-configmap.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * authentication/certificates/service-serving-certificate.adoc
+
+[id="add-service-certificate-configmap-{context}"]
+= Add a service certificate to a ConfigMap
+
+To secure communication to services that use a ConfigMap, 
+have the cluster generate a signed serving certificate and key pair into the
```

Comment:
This is incorrect. (the wording also implies that the service uses the configmap) 
What you are injecting into the configMap is just the CA certificate. The purpose for this is so that a pod seperate from a service can get access to the CA certificate, in order to do TLS against a Service that is using a service serving cert.

Reply:
I've updated the introductory text to read:

> A Pod can obtain a reference to the service CA certificate, so that the Pod 
> can securely communicate with a service. To accomplish this communication, 
> have the cluster insert the service CA certificate into the ConfigMap by
> adding the  
> `service.beta.openshift.io/inject-cabundle=true` 
> annotation on your ConfigMap. 
> 
> The service CA certificate is stored in the ConfigMap as `service-ca.crt`.